### ISSUE TITLE
Unify sample data and usage helpers

### ISSUE THREAD

**Issue by pmbrull** [open-metadata/OpenMetadata#1049]
We have quite similar helpers defined in `metadata/ingestion/utils/helpers` and in `metadata/ingestion/source/sample_data.py`.

It'd be interesting to reorganise a bit the `sample_data` code to pick all functionalities from `helpers`. We'll need to redefine a bit the parameters & signature.

Review where the different functions are being used to unify the arguments' definitions and test it out running `metadata ingest -c pipelines/sample_data.json`. No errors nor failures should appear.

**Comment by TeddyCr** [comment]
Hey @pmbrull I would love to tackle this one 🙂

**Comment by TeddyCr** [comment]
Hey @pmbrull looks like I am getting a new error when trying to run the project today. I am following [these steps](https://docs.open-metadata.org/open-source-community/developer/build-code-run-tests#create-a-distribution-packaging) to build a distribution and I am getting the below error when running `cd dist && mvn package` -- I just merged the latest changes on `main` into my branch. Yesterday night I was able to build the distribution without any problems.

Have encountered this before?

```
...
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (/path/OpenMetadata/dist). Please verify you invoked Maven from the correct directory. -> [Help 1]
...
```
